### ISSUE TITLE
Consider ways to get fiat currency to ETH for easy entry to crypto

### ISSUE THREAD

**Issue by px3l** [strange-labs-uk/ethereum-lottery#8]
Like coinbase makes it easy, we need it easy enough for new comers to invest in crypto through the lottery. Maybe this is a simple guide on the customer facing page

**Comment by brtknr** [comment]
Yes, not an awful lot we can do here other than point out popular places where people can buy ether.
